packages/esm-patient-chart-app/src/clinical-views/utils/helpers.ts

```diff
@@ -20,21 +21,24 @@ export function launchEncounterForm(
   intent: string = '*',
   patientUuid?: string,
 ) {
-  launchWorkspace('patient-form-entry-workspace', {
-    workspaceTitle: form?.name,
-    mutateForm: onFormSave,
-    formInfo: {
-      encounterUuid,
-      formUuid: form?.uuid,
-      patientUuid: patientUuid,
-      visit: visit,
-      additionalProps: {
-        mode: action === 'add' ? 'enter' : action,
-        formSessionIntent: intent,
-        openClinicalFormsWorkspaceOnFormClose: false,
+  if (!visit) {
```

Comment:
Could we put this behind a configuration option so OHRI can continue to use these without visits? (The configuration option should default to using visits).